ehitchhiking-ui/src/app/shared/components/active-trip/active-trip.component.ts

```diff
@@ -0,0 +1,37 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {StarClickMeta} from '../rating/starClickMeta';
+import {ActiveTrip} from './active-trip';
+
+@Component({
+	selector: 'app-active-trip',
+	templateUrl: './active-trip.component.html',
+	styleUrls: ['./active-trip.component.sass'],
+})
+export class ActiveTripComponent implements OnInit {
+	constructor() {}
+
+	ngOnInit() {}
+
+	@Input() trip: ActiveTrip;
+	isRatingEditorVisible: boolean;
+
+	makeFavorite() {
+		this.trip.isFavorite = !this.trip.isFavorite;
+	}
+
+	toggleRating() {
+		this.isRatingEditorVisible = !this.isRatingEditorVisible;
+	}
+
+	rateTrip(clickObj: StarClickMeta): void {
```

Comment:
We should not set rating for non completed trips

Reply:
removed